core/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -721,4 +723,19 @@ public static CircuitBreakerService createCircuitBreakerService(Settings setting
     BigArrays createBigArrays(Settings settings, CircuitBreakerService circuitBreakerService) {
         return new BigArrays(settings, circuitBreakerService);
     }
+
+    /**
+     * Generate Custom Name Resolvers list based on a Discovery Plugins list
+     * @param discoveryPlugins Discovery plugins list
+     */
+    private List<NetworkService.CustomNameResolver> generateCustomNameResolvers(List<DiscoveryPlugin> discoveryPlugins) {
```

Comment:
nit: generate implies creation, but really this is only pulling them all together. Perhaps just `get`?